src/test/resources/features/Tasks/Projects.feature

```diff
@@ -48,7 +48,7 @@ Feature: Tasks - Projects
     And The placeholder Filter by project should be displayed
     And The clear button is disappeared from the Filter by project field
 
-  @ignored
+  @task
```

Comment:
there is already `@task` for the whole Feature file, no need to add it here again